src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
My intent was:
* If `anti_lmr7` is successful, accept both patches
* If it fails, do additional experiments to understand the cause of failure

But now I am told that simply from reading code and its code style, without deeply understanding it, I should judge.  So fine, I'll accept anything which gains Elo and stop attempting to do code review.